tracer/test/Datadog.Trace.TestHelpers/UnixDomainSocketConfig.cs

```diff
@@ -7,10 +7,11 @@ namespace Datadog.Trace.TestHelpers
 {
     public class UnixDomainSocketConfig
     {
-        public UnixDomainSocketConfig(string traces, string metrics)
+        public UnixDomainSocketConfig(string traces, string metrics, MockTracerAgent.AgentConfiguration configuration = null)
```

Comment:
nit: I probably would have added this to the `MockTracerAgent` and helper functions as an extra param rather than here, but I haven't looked at whether that's actually easier 🙂

Reply:
Good idea, I did that in the new PR #3152